### PR TITLE
Add __version__ string and tweak versioning scheme

### PR DESCRIPTION
**Pull request by ddfisher**
Going forward, we will increment the version and append `-dev` to the version string in between released versions.

### CODE REVIEW

File: typed_ast/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-
+__version__ = "1.1.0"
```
Comment (gvanrossum) [inline typed_ast/__init__.py]: It will have to be 1.1.1 though. PyPI won't let you upload a new version.

Comment (ddfisher) [inline typed_ast/__init__.py]: I'm not trying to change the version as part of this PR -- versions bumps have their own commits.

Comment (gvanrossum) [inline typed_ast/__init__.py]: The tradition we use in mypy is that immediately after tagging a release we add a new commit that bumps the version to <next-version>-dev. So here you would first submit and merge a diff that bumps the version to 1.1.1-dev, and once that's landed you'd merge this one (but with 1.1.1-dev in there somewhere).

Comment (ddfisher) [inline typed_ast/__init__.py]: Yeah, I know that's how mypy does it.  I figured that was too heavy-weight for typed_ast, which often goes just one or two commits between releases, but I guess it's not actually much additional work.  Do you think it's worth switching to that versioning scheme?

**Comment by gvanrossum** [comment]
Yes.

**Comment by webknjaz** [comment]
@ddfisher is there any reason for not using `setuptools-scm`? It simplifies version management a lot by looking up a semver tagged commit @ git history

**Pull request merged**